src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,155 @@
+      setError('');
+
+      // eslint-disable-next-line
+      const { Title: movieTitle, Plot: description, Poster: imgUrl, imdbID: imdbId } = movieFromServer;
+
+      setMovie({
+        title: movieTitle,
+        description,
+        imgUrl,
+        imdbUrl: `https://www.imdb.com/title/${imdbId}`,
+        imdbId,
+      });
+    } else {
+      setError(`Can&apos;t find a movie with such a title`);
+    }
+  };
+
+  const setUpMovieTitle = (event) => {
+    if (title) {
+      clearState();
+    }
+
+    setTitle(event.target.value);
+  };
+
+  const addMovieToList = () => {
+    clearState();
+    if (isMovieInList()) {
+      setError(`can&apos;t add, this movie is on the list`);
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
+      return;
+    }
+
+    if (movie.title) {
+      onAdd(movie);
+    }
+  };
+
+  const isMovieInList = () => movies.some(curMovie => (
+    curMovie.imdbId === movie.imdbId
+  ));
+
+  const clearState = () => {
+    setTitle('');
+    setMovie(moviePattern);
+    setIsMovieCard(false);
+    setError('');
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
+              value={title}
+              onChange={event => setUpMovieTitle(event)}
+            />
+          </div>
+
+          <p className={classnames({
+            help: true,
+            's-danger': true,
+            hide: error.length === 0,
+          })}
           >
-            Find a movie
-          </button>
+            {error}
+          </p>
+
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={searchMovie}
+            >
+              Find a movie
+            </button>
+          </div>
+
+          <div className="control">
+            <button
+              type="button"
+              className={classnames({
+                button: true,
+                'is-primary': true,
+                hide: !isMovieCard,
+              })}
+              onClick={addMovieToList}
+            >
+              Add to the list
+            </button>
+          </div>
         </div>
+      </form>
+
+      <div className={classnames({
+        container: true,
+        hide: error.length > 0 || !isMovieCard,
```

Comment:
```suggestion
        hide: !!error || !isMovieCard,
```
Use conditional rendering